app/styles/atoms/progress-bar.less

```diff
@@ -0,0 +1,41 @@
+.oss-progress-bar--wrapper {
+  min-width: 220px;
```

Comment:
Why define a min-width ? 🤔

Reply:
Removed & will be managed from parent ( as seen in SB story )